src/get-start-and-end-commands.js

```diff
@@ -64,7 +64,7 @@ async function _prepareCommand({
     rimraf(path.join(appPath, 'yarn.lock'))
   ]);
 
-  return `node ${cpr} ${appPath} .`;
+  return `node "${cpr}" "${appPath}" .`;
```

Comment:
technically, for my use case, it'd be enough to just wrap `cpr` with a double-quotes. However, since it seems to be a recommended approach, I've also wrapped the `appPath`.